src/server/encryption.rs

```diff
@@ -59,7 +57,7 @@ impl Cryptor {
         } = payload;
 
         let mut nonce_buf = [0u8; aead::NONCE_LEN];
-        self.rng
+        rand::SystemRandom::new()
```

Comment:
This change seems fine but orthogonal to the purpose of the PR. Was it required for some reason?

It's fine to leave in -- I'm just curious.